testing/common_testing_util.py

```diff
@@ -1,7 +1,16 @@
+import os
```

Comment:
I don't think "os" is used in the changes in this file.

Reply:
Removed. 👍